packages/amplication-server/src/main.ts

```diff
@@ -52,7 +54,16 @@ async function bootstrap() {
     app.enableCors();
   }
 
-  await app.listen(process.env.PORT || 3000);
+  const appMode = process.env.APP_MODE || "http";
+  if (appMode === "https") {
+    const httpsServer = https.createServer(httpsOptions, app.getHttpServer());
+    httpsServer.listen(443);
+    console.log("HTTPS server is running on port 443");
```

Comment:
Please use Nestjs logger instead of console.log.

More details here https://docs.nestjs.com/techniques/logger#dependency-injection